src/statusbar.ts

```diff
@@ -37,12 +37,25 @@ function colorValue(
     section: string,
     currentConf: vscode.WorkspaceConfiguration
 ): string {
+    const configSection = currentConf.inspect<string>(section);
+
+    util.assertIsDefined(
+        configSection,
+        () => `Expected config section "${section}" to be defined!`
+    );
+
     const { defaultValue, globalValue, workspaceFolderValue, workspaceValue } =
-        currentConf.inspect(section);
-    return (workspaceFolderValue ||
-        workspaceValue ||
-        globalValue ||
-        defaultValue) as string;
+        configSection;
+
+    const value =
+        workspaceFolderValue || workspaceValue || globalValue || defaultValue;
+
+    // Current behavior is to assert that this is a string even though it may
+    // not be. Maintaining current behavior for the moment but we should
+    // eventually do an assertion here or allow the function to return
+    // undefined.
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion, @typescript-eslint/no-unnecessary-type-assertion
+    return value!;
```

Comment:
Is this `!` after the variable intentional? If so, what does it do? If I try to evaluate something like that in the console directly (like `myVar!`), I get an exception that `!` is an unexpected token. I'm guessing this is just something that works here that I'm unaware of.